src/tls.rs

```diff
@@ -136,7 +144,7 @@ impl TlsConfigBuilder {
 
         let mut config = ServerConfig::new(NoClientAuth::new());
         config
-            .set_single_cert(cert, key)
+            .set_single_cert_with_ocsp_and_sct(cert, key, self.ocsp_resp, Vec::new())
```

Comment:
If these vecs are empty, is it the same as just calling `set_single_cert`?

Reply:
yeah, by [`ServerConfig::set_single_cert_with_ocsp_and_sct`](https://docs.rs/rustls/0.18.1/rustls/struct.ServerConfig.html#method.set_single_cert_with_ocsp_and_sct) doc: 
`ocsp is a DER-encoded OCSP response. Ignored if zero length. scts is an SignedCertificateTimestampList encoding (see RFC6962) and is ignored if empty.`